common.sh

```diff
@@ -71,99 +99,165 @@ log()
         fi
     fi
 
-    echo "$SEVERITY $COMPONENT: $1"
+    echo "$SEVERITY $COMPONENT: $1" >&2
```

Comment:
I believe they should. Just example: you create function get_list_.... and uses log_* inside. In that case, you will get some expected output mixed with logs. All logs like log_high_risk, log_warning, log_info, .... should be really on stderr.